vassal-app/src/main/java/VASSAL/tools/ArchiveWriter.java

```diff
@@ -267,26 +267,31 @@ public void addFile(String fileName, byte[] content) {
    * Saves the archive, prompting for a name only if none has ever been provided.
    * @throws IOException IOException
    */
-  public void save() throws IOException {
-    save(false);
+  public boolean save() throws IOException {
+    return save(false);
   }
 
   /**
    * Saves the archive, prompting for a name only if none has ever been provided.
    * @param notifyModuleManager If true, notifies Module Manager that the save has occurred
    * @throws IOException IOException
    */
-  public void save(boolean notifyModuleManager) throws IOException {
-    if (isTempArchive) saveAs(notifyModuleManager);
-    else write(archive, notifyModuleManager);
+  public boolean save(boolean notifyModuleManager) throws IOException {
+    if (isTempArchive) {
+      return saveAs(notifyModuleManager);
+    }
+    else {
+      write(archive, notifyModuleManager);
+    }
+    return false;
```

Comment:
Shouldn't we return `true` after `write()`?

Reply:
Is this the cause of the problem noted here? http://www.vassalengine.org/forum/viewtopic.php?p=67818#p67818

Reply:
Yep, had to make a new PR for it, so see #891